lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -267,15 +302,26 @@ def self.bump_version_release
           return verified_version
         end
 
-        # Update the `build.gradle` file with new `versionName` and `versionCode` values, both or the `defaultConfig` and `vanilla` flavors
+        # Update the `version.properties` file with new `versionName` and `versionCode` values
         #
-        # @param [Hash] new_version_beta The version hash for the beta (vanilla flavor), containing values for keys "name" and "code"
-        # @param [Hash] new_version_alpha The version hash for the alpha (defaultConfig), containing values for keys "name" and "code"
-        # @env HAS_ALPHA_VERSION If set (with any value), indicates that the project uses `vanilla` flavor.
+        # @param [Hash] new_version_beta The version hash for the beta, containing values for keys "name" and "code"
+        # @param [Hash] new_version_alpha The version hash for the alpha , containing values for keys "name" and "code"
         #
-        def self.update_versions(new_version_beta, new_version_alpha)
-          self.update_version(new_version_beta, ENV['HAS_ALPHA_VERSION'].nil? ? 'defaultConfig' : 'vanilla {')
-          self.update_version(new_version_alpha, 'defaultConfig') unless new_version_alpha.nil?
+        def self.update_versions(app, new_version_beta, new_version_alpha)
+          if properties_file_exists
+            self.update_version(new_version_beta, ENV['HAS_ALPHA_VERSION'].nil? ? 'defaultConfig' : 'vanilla {')
+            self.update_version(new_version_alpha, 'defaultConfig') unless new_version_alpha.nil?
+            return
+          end
```

Comment:
Once again logic seems inverted to me here? (at that point, I'm guessing, probably at every place where this is used too)

Reply:
Yep you're right on this one